tools/as/as.cpp

```diff
@@ -79,6 +84,15 @@ int main(int argc, char** argv) {
             return 1;
           }
         } break;
+        case '-': {
+          // Long options
+          if (0 == strcmp(argv[argi], "--version")) {
+            printf("%s\r\n", kBuildVersion);
```

Comment:
Nope, `\n` is the platform-specific newline everywhere.  You only need `\r` for cross-platform reading and writing (eg, reading Windows text files on Unix).

I agree about `printf()`, but that's a separate story and can wait for now.